customer/interfaces/dtocustomer/dto.go

```diff
@@ -0,0 +1,22 @@
+package dtocustomer
```

Comment:
this dto obkects are just used in the graphql package? I think in other places we therefore moved it as subpackage to the graphql folder?